storage/spanstore/memory/memory.go

```diff
@@ -201,22 +210,14 @@ func (m *Store) validSpan(span *model.Span, query *spanstore.TraceQueryParameter
 	}
 	spanKVs := m.flattenTags(span)
 	for queryK, queryV := range query.Tags {
-		keyValueFoundAndMatches := false
-		// (NB): we cannot find the KeyValue.Find function because there can be multiple tags with the same key
-		for _, keyValue := range spanKVs {
-			if keyValue.Key == queryK && keyValue.AsString() == queryV {
-				keyValueFoundAndMatches = true
-				break
-			}
-		}
-		if !keyValueFoundAndMatches {
+		// (NB): we cannot find the KeyValues.FindKey function because there can be multiple tags with the same key
```

Comment:
we cannot use...

Reply:
Yes added back note from master. Had to write local helper function instead.

Reply:
I meant changing the comment "we cannot **use**", not "find"

Reply:
oh lol didn't read carefully